### ISSUE TITLE
Multiple selections "undo"

### ISSUE THREAD

**Issue by cesarmtnez** [dail8859/BetterMultiSelection#10]
It will be great if your plugin would be able to remember the multiple focus after undoing (Ctrl + Z). Now, after applying 'undo', the focus switches to normal focus on a single line.

If you want to better understand what I mean, you can try that in Sublime Text.

**Comment by pidgeon777** [comment]
I also second this great request, of which I also thought. It would almost be the definitive consecration of your plugin, in my opinion.

**Comment by dail8859** [comment]
Yeah I also agree it is an excellent idea. I'm just not sure how easy it will be to do though. Scintilla provides a way to register custom undo/redo actions and I haven't had a chance to try it out much yet.

**Comment by pidgeon777** [comment]
I don't know if it could help, but with the latest released version of Notepad++, Scintilla also got finally updated too, after many years. It should be interesting to test your released plugin also to check if compatibility has been maintained. I'm planning to upgrade from Notepad++ 7.6.3 to 7.7, thus I'll have a chance to do that over the next weeks.
